cube_builder/business.py

```diff
@@ -62,15 +62,20 @@ def create(cls, params: dict):
         for cube in cubes:
             # save bands
             for band in params['bands']:
+                if band == 'cnc' and 'WARPED' in cube.id:
```

Comment:
find it interesting to put another check? -> `or (band == 'quality' and not 'WARPED' in cube.id)`, because we are not generating quality in `blend`

Reply:
sure